client/src/About.jsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const About = (props) => {
```

Comment:
Have you installed [React dev tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)?  It will let you directly inspect props & state for your components in real time.  Very handy for debugging.